### ISSUE TITLE
Unrecognized model_output parameter value: predict_proba with TreeExplainer and scikit RandomForestClassifier

### ISSUE THREAD

**Issue by SerissaDahoe** [shap/shap#1419]
Hi,

I was trying to use the TreeExplainer on a scikit Random Forest model, with the predict_proba function, even mentioned as an example in the docs here:
<img width="708" alt="image" src="https://user-images.githubusercontent.com/39773727/92715712-23200d80-f35e-11ea-838c-4c8f8287b6cf.png">

I'm using below code:
```
import shap
from sklearn import datasets
from sklearn.ensemble import RandomForestClassifier

iris = datasets.load_iris()
X = iris.data[:, :2]  # we only take the first two features.
y = iris.target
model = RandomForestClassifier()
model.fit(X, y);
y_pred = model.predict_proba(X)

explainer = shap.TreeExplainer(model, feature_perturbation='interventional', 
                               model_output="predict_proba", data=X)
shap_values = explainer.shap_values(X)
```

And getting the following exception:
<img width="735" alt="image" src="https://user-images.githubusercontent.com/39773727/92715955-74c89800-f35e-11ea-9a3c-88f72e9c7666.png">

Has anyone else had a similar issue, or knows how to fix it?

**Comment by roye10** [comment]
yes! similar issue here. Just used model_output = 'probability'. Otherwise no clue how to fix this.

**Comment by SerissaDahoe** [comment]
Hmm strange, I suppose that is the only solution then, thanks!

**Comment by slundberg** [comment]
I updated the error message to help here. The issue is that 'predict_proba' should explain a multi-output model with two outputs, while 'probability' is just the probability of the 1 class. We could support 'predict_proba' with some copying, but it is not done yet...
